src/app/components/MediaLoader/index.test.tsx

```diff
@@ -191,19 +195,31 @@ describe('MediaLoader', () => {
       await act(async () => {
         render(
           <MediaPlayer
-            blocks={onDemandTvBlocks as MediaBlock[]}
-            pageIdentifierOverride="hindi.bbc_hindi_tv.tv.w172zm8b4tlpzxh.page"
+            blocks={onDemandTvBlocksWithOverrides as MediaBlock[]}
             embedded
           />,
           { service: 'hindi' },
         );
       });
 
-      expect(buildConfigSpy).toHaveBeenCalledWith(
+      expect(buildConfigSpy.mock.calls[0][0]).toEqual(
         expect.objectContaining({
           counterName: 'hindi.bbc_hindi_tv.tv.w172zm8b4tlpzxh.page',
         }),
       );
+      expect(buildConfigSpy.mock.calls[0][0].blocks).toEqual(
+        expect.arrayContaining([
+          {
+            model: {
+              language: 'hi',
+              pageIdentifierOverride:
+                'hindi.bbc_hindi_tv.tv.w172zm8b4tlpzxh.page',
+              pageTitleOverride: 'दुनिया',
+            },
+            type: 'mediaOverrides',
+          },
+        ]),
+      );
     });
 
     it('should use the pageIdentifier from the EventTrackingContext', async () => {
```

Comment:
Does this test description need to be updated now?

Reply:
I've updated this description to convey that we're defaulting to
`pageIdentifier` from the `EventTrackingContext` when overrides are not present.